src/components/LoginForm/LoginForm.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { ViewIcon, ViewOffIcon } from '@chakra-ui/icons';
+import {
+  Button,
+  FormControl,
+  FormHelperText,
+  FormLabel,
+  Input,
+  InputGroup,
+  InputRightElement,
+  Stack
+} from '@chakra-ui/react';
+export const LoginForm = () => {
+  const [show, setShow] = React.useState(false);
+  const handleClick = () => setShow(!show);
+
+
+  const {
+    register,
+    handleSubmit,
+    watch,
+    reset,
+    formState: { errors }
+  } = useForm({
+    reValidateMode: 'onSubmit',
+    mode: 'onTouched'
+  });
+
+  const email = watch('email');
+  const password = watch('password');
+  console.log(watch());
+  const isValid = email && password;
+
+  const onSubmit = (data) => {
+    console.log(data);
+    reset();
+  };
+
+  const onError = (error) => {
+    console.log(error);
+  };
+
+  return (
+    <Stack
+      width="500px"
+      p="4"
+      boxShadow="xl"
+      borderRadius="4px"
+      background="white"
+    >
+      <form onSubmit={handleSubmit(onSubmit, onError)}>
+        <FormControl>
+          <FormLabel htmlFor="email">Email</FormLabel>
+          <Input
+            id="email"
+            type="email"
+            {...register('email', { required: true })}
+          />
+          {errors && errors.email && (
```

Comment:
OPTIONAL: maybe will be a good improvement to move the logic of getting an error message to function and then reuse it under each field.